spark3/src/main/java/org/apache/iceberg/spark/SparkCatalog.java

```diff
@@ -164,13 +183,24 @@ public SparkTable createTable(Identifier ident, StructType schema,
   public StagedTable stageCreate(Identifier ident, StructType schema, Transform[] transforms,
                                  Map<String, String> properties) throws TableAlreadyExistsException {
     Schema icebergSchema = SparkSchemaUtil.convert(schema);
+    TableIdentifier tableIdentifier = buildIdentifier(ident);
     try {
-      return new StagedSparkTable(icebergCatalog.newCreateTableTransaction(
-          buildIdentifier(ident),
-          icebergSchema,
-          Spark3Util.toPartitionSpec(icebergSchema, transforms),
-          properties.get("location"),
-          Spark3Util.rebuildCreateProperties(properties)));
+      Transaction transaction;
+      if (isPathIdentifier(ident)) {
+        transaction = tables.newCreateTableTransaction(
+            tableIdentifier.name(),
```

Comment:
This should be `tableIdentifier.location()` right?

Reply:
yes, fixed.